examples/user_guide/14-Data_Pipelines.ipynb

```diff
@@ -152,7 +150,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Since we have a handle on the ``Stream`` we can now send events to it and watch the plot above update, let's start by setting the ``rolling_window=50``. "
+    "The ``rolled_dmap`` is another DynamicMap which defines a simple two step pipeline, which calls the original callback when the ``symbol`` changes and reapplies the ``rolling`` operation when the slider value changes. Since the widget's value is now linked to the plot via a ``Stream`` we can display the widget and watch the plot update:"
```

Comment:
```suggestion
    "The ``rolled_dmap`` is another DynamicMap that defines a simple two-step pipeline, which calls the original callback when the ``symbol`` changes and reapplies the ``rolling`` operation when the slider value changes. Since the widget's value is now linked to the plot via a ``Stream`` we can display the widget and watch the plot update:"
```